### PR TITLE
[ocm-groups] introduce new integration

### PR DESCRIPTION
**Pull request by maorfr**
covers https://issues.redhat.com/browse/APPSRE-1382

This integration adds the ability to manage OpenShift groups via OCM.

### CODE REVIEW

File: utils/ocm.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        ocm_offline_token = ocm_info.get('offlineToken')
+        if ocm_offline_token is None:
+            self.ocm_map[ocm_name] = False
+        else:
+            url = ocm_info['url']
+            token = secret_reader.read(ocm_offline_token, self.settings)
+            self.ocm_map[ocm_name] = \
+                OCM(url, access_token_client_id, access_token_url, token)
+
+    def cluster_disabled(self, cluster_info):
```
Comment (jmelis) [inline utils/ocm.py]: nitpick:

```python
        try:
            integrations = cluster_info['disable']['integrations']
        except (KeyError, TypeError):
            pass

        return self.calling_integration.replace('_', '-') in integrations
```

Comment (jmelis) [reply utils/ocm.py]: on second though I don't like my suggestion

**Comment by jmelis** [comment]
LGTM

**Comment by maorfr** [comment]
added docstrings as described in our style guide.

@apahim thanks for pointing at https://github.com/NilsJPWerner/autoDocstring!

**Pull request merged**